src/utils/ballerinaUtils.ts

```diff
@@ -0,0 +1,29 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IActionContext } from "@microsoft/vscode-azext-utils";
+import * as vscode from 'vscode';
+import { localize } from '../localize';
+import { cpUtils } from './cpUtils';
+
+export namespace ballerinaUtils {
+    const ballerinaCommand: string = 'bal';
+
+    export async function validateBallerinaInstalled(context: IActionContext): Promise<void> {
+        try {
+            await cpUtils.executeCommand(undefined, undefined, ballerinaCommand, '--version');
+        } catch (error) {
+            const message: string = localize('ballerinaNotFound', 'Failed to find "bal", please ensure that the bal bin directory is in your system path.');
```

Comment:
I know Java doesn't do this, but I think it'd be helpful to link the Ballerina download page in either the message or have an "open" button that comes up with the error message.

Reply:
> I know Java doesn't do this, but I think it'd be helpful to link the Ballerina download page in either the message or have an "open" button that comes up with the error message.

Great suggestion, will add this ;)

Reply:
Done